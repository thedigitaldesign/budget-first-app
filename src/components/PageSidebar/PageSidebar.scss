.sidebar-menu {
  background-color: #303641;
  color: #fff;
  width: 20rem;
  max-width: 20rem;

  .sidebar-menu-inner {
    position: relative;

    .info-card {
      .info-card-text {
        
      }
    }

    .page-logo {

    }

    .primary-nav {

    }
  }
}