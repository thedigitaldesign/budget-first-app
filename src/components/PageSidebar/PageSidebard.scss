.page-sidebar {
  width: 16.75rem;
}