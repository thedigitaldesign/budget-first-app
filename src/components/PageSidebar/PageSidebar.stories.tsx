import React from 'react'

// Packages
import { storiesOf } from '@storybook/react'

// Component
import PageSidebar from './PageSidebar'

storiesOf('Components/PageSidebar', module).add('Page Sidebar', () => {
  return(
    <div>
      <PageSidebar />
    </div>
  )
})
