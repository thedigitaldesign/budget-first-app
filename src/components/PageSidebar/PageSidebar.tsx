import React, { Component } from 'react'

// Styles
import './PageSidebar.scss'

// Components
import InfoCard from './InfoCard/InfoCard'
import PageLogo from './PageLogo/PageLogo'
import PrimaryNav from './PrimaryNav/PrimaryNav'

export default class PageSidebar extends Component {
  render() {
    return (
      <aside className="sidebar-menu column is-2 is-hidden-mobile">
        <PageLogo />
        <InfoCard />
        <PrimaryNav />
      </aside>
    )
  }
}
