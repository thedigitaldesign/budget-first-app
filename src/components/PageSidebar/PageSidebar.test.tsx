import React from 'react'

// Packages
import { shallow } from 'enzyme'

// Components
import PageSidebar from './PageSidebar'

describe('PageSidebar', () => {
  const sidebar = shallow(<PageSidebar />)

  it('renders correctly', () => {
    expect(sidebar).toMatchSnapshot()
  })

  it('contains an InfoCard component', () => {
    expect(sidebar.find('InfoCard').exists()).toBe(true)
  })

  it('contains an PageLogo component', () => {
    expect(sidebar.find('PageLogo').exists()).toBe(true)
  })

  it('contains an PrimaryNav component', () => {
    expect(sidebar.find('PrimaryNav').exists()).toBe(true)
  })
})
