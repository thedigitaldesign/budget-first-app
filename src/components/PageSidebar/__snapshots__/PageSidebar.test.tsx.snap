// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageSidebar renders correctly 1`] = `ShallowWrapper {}`;
